### ISSUE TITLE
Show info box content only on scroll-over

### ISSUE THREAD

**Issue by mariobehling** [fossasia/eventyay-talk#16]
### Problem and impact

Many of the static [i] boxes include common sense information. Also for organizers who use the system continuously the [i] boxes could get annoying. Therefore change the instead of showing an entire box show an [i] icon that displays the information on scroll-over or clicking. The icon should be displayed behind the page title.

![Screenshot from 2024-01-19 18-08-47](https://github.com/fossasia/eventyay-talk/assets/1583873/736f74ab-d2b5-45ed-a36e-43bf1f07d640)

**Comment by Aryan20** [comment]
Hi! I am working on this and want some feedback.
How does this look?

![image](https://github.com/fossasia/eventyay-talk/assets/34372791/0aec1f60-ed6b-46fd-b785-d6258283f2b1)


![image](https://github.com/fossasia/eventyay-talk/assets/34372791/ba9a8f22-1553-4da9-b045-b359133dc07b)

**Comment by Aryan20** [comment]
Also, can you provide me with the pages where we need to make this change? As in some places, it might be better to keep it as it is right?

**Comment by mariobehling** [comment]
Yes, you are on the right track and correct on some pages it might be better to keep it as. I think we might have to go step by step here as there are a number of pages with infos. Let me go through this. I definitely appreciate that you think along what is best for the project.

**Comment by Aryan20** [comment]
Thanks :D
Sure, will wait for your reply. Till then will work on making it more responsive for mobile devices.
